notebooks/rag.ipynb

```diff
@@ -1222,7 +1140,11 @@
     "    batch_size=128,\n",
     "    num_cpus=1,\n",
     "    compute=ActorPoolStrategy(size=6),\n",
-    ").count()"
+    ").materialize()\n",
```

Comment:
`.count()` will not execute the function `StoreResults` indefinitely. I referred to [this blog](https://www.anyscale.com/blog/rag-at-scale-10x-cheaper-embedding-computations-with-anyscale-and-pinecone), and used `materialize()` instead.